rllib/agents/pg/__init__.py

```diff
@@ -1,6 +1,11 @@
 from ray.rllib.agents.pg.pg import PGTrainer, DEFAULT_CONFIG
+from ray.rllib.agents.pg.tf_pg_policy import tf_pg_loss, \
+    post_process_advantages
+from ray.rllib.agents.pg.torch_pg_policy import torch_pg_loss
 from ray.rllib.utils import renamed_agent
 
+# `Agent`=Legacy: Use PGTrainer instead.
 PGAgent = renamed_agent(PGTrainer)
```

Comment:
Btw we can probably start removing these, they've been deprecated for a while now.